requirements.txt

```diff
@@ -0,0 +1,3 @@
+blinker==1.4
+ts3==0.6.7
```

Comment:
I think that we can unfreeze dependencies.

Reply:
fixed